plugins/feeds/public/panunit42_github_iocs.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from dateutil import parser
+from datetime import datetime, timedelta
+from core.feed import Feed
+from core.errors import GenericYetiError
+from core.observables import Observable
+from core.observables import Hash, Url, Hostname, Ip, MacAddress, Email, Certificate
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+class PanUnit42GithubIocs(Feed):
+
+    default_values = {
+        'frequency': timedelta(hours=24),
+        'name': 'PanUnit42GithubIocs',
+        'source': 'https://api.github.com/repos/pan-unit42/iocs/commits',
+        'description': 'Get Iocs from Pan-Unit42 GitHub Iocs repo',
+    }
+    refs = {
+        'MacAddress': MacAddress,
+        'Hash': Hash,
+        'Url': Url,
+        'Ip': Ip,
+        'FileHash-SHA1': Hash,
+        'Hostname': Hostname,
+        'Email': Email,
+    }
+
+    blacklist = ('Makefile', 'LICENSE', 'README.adoc')
+    blacklist_domains = ('technet.microsoft.com', 'cloudblogs.microsoft.com', 'capec.mitre.org',  'attack.mitre.org', 'securelist.com', 'blog.avast.com')
```

Comment:
I'd set this as a global variable at the top of the file (e.g. outside the class): 

```python
BLACKLIST_DOMAINS = [
    'technet.microsoft.com',
    'cloudblogs.microsoft.com',
    'capec.mitre.org',
    'attack.mitre.org',
    'securelist.com',
    'blog.avast.com',
]
```

Same with the file blacklist

Reply:
the blocklist is used inside of the update_github